src/js/visual/configurator-csr.js

```diff
@@ -1675,10 +1675,18 @@ ripe.ConfiguratorCsr.prototype._onInitialsEvent = function(self, initials, engra
 /**
  * @ignore
  */
-ripe.ConfiguratorCsr.prototype._onInitialsExtraEvent = function(self, initialsExtra, params) {
+ripe.ConfiguratorCsr.prototype._onInitialsExtraEvent = function(self, initialsExtra = {}, params) {

```

Comment:
Add only the default values to the optional parameters. In this case it would be:
```suggestion
ripe.ConfiguratorCsr.prototype._onInitialsExtraEvent = function(self, initialsExtra, params = [}) {
```